test/suites/common/setup.yaml

```diff
@@ -103,4 +103,5 @@ spec:
         --set clusterName=$(params.cluster-name) \
         --set clusterEndpoint=$(cat /root/.kube/config | grep server | awk '{print $2}') \
         --set aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-$(params.cluster-name) \
+        --set controller.batchIdleDuration=10s \
```

Comment:
do we need this? Make me a little uneasy to set this to something that's not the default.

Reply:
I was seeing issues where pods would get batched into two separate nodes when deploying with deployment controller which made the tests unreliable.

Reply:
Not sure if there's a better way to handle this since we are relying on strict node count comparison for things like Expiration testing